irc.html

```diff
@@ -30,338 +32,319 @@
 </head>
 
 <body style="font-family: 'Montserrat', sans-serif;">
-<header class="header-global">
-    <!--nav bar goes here-->
-    <div id="navbar">
-
-    </div>
-</header>
+    <main>
+        <header class="header-global">
+            <!--nav bar goes here-->
+            <div id="navbar"></div>
+        </header>
+    </main>
```

Comment:
@YoshithaRathnayake  Normally we use the `main` tag to wrap the non-repeating unique set of elements in a page. So it is not a good practice to put the navbar inside the main tag. 

Btw this alignment issue is related to some kind of overflow property of the navbar. Try using `overflow-hidden` class with the navbar.

Reply:
Ok @Piumal1999

Reply:
> @YoshithaRathnayake Normally we use the `main` tag to wrap the non-repeating unique set of elements in a page. So it is not a good practice to put the navbar inside the main tag.
> 
> Btw this alignment issue is related to some kind of overflow property of the navbar. Try using `overflow-hidden` class with the navbar.

@Piumal1999 I changed it but can't reproduce the bug from overflow hidden

Reply:
> > @YoshithaRathnayake Normally we use the `main` tag to wrap the non-repeating unique set of elements in a page. So it is not a good practice to put the navbar inside the main tag.
> > Btw this alignment issue is related to some kind of overflow property of the navbar. Try using `overflow-hidden` class with the navbar.
> 
> @Piumal1999 I changed it but can't reproduce the bug from overflow hidden

No sorry @Piumal1999 the bug is fixing but when I put overflow hidden the nav dropdowns are hiding

Reply:
Resolved @Piumal1999 the problem was the width of the join us dropdown so decreased the width by dropdown-menu-lg